.github/workflows/ci-app-pr-environment-checks.yml

```diff
@@ -1,4 +1,4 @@
-name: CI App PR Environment Update
+name: CI App PR Environment Checks
```

Comment:
Renamed since this is no longer just updating the PR preview env, but also running e2e (and a11y) checks